tasks/configure.yml

```diff
@@ -2,10 +2,11 @@
 - name: Ensure ccache cache directory exists
   become: true
   file:
-    path: "{{ ccache_cache_dir }}"
+    path: "{{ ccache_config_options['cache_dir'] }}"
     state: directory
     owner: "{{ ccache_user }}"
     mode: "0755"
+  when: ccache_config_options['cache_dir'] is defined
```

Comment:
Why do we have that "Ensure ccache cache directory exists" task in the first place? If that directory doesn't exist, ccache should be able to create it itself.

Reply:
Ok, let's remove the task entirely. Fixed in 308c619.